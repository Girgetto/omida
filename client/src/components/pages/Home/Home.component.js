import React from 'react';

const Home = ({ className }) => {
  return (
    <div className={className}>
      <h1>Hotel Finder</h1>
      <div>
        <input/>
        <button>Find</button>
      </div>
    </div>
  );
};

export default Home;
