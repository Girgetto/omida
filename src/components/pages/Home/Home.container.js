import { connect } from 'react-redux'
import Login from './Home.styled'

export default connect()(Login)
