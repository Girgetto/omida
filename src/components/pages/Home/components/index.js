import FirstSection from './FirstSection';
import SecondSection from './SecondSection';
import OurMission from './OurMission';
import FirstRow from './FirstRow';
import SecondRow from './SecondRow';
import ThirdRow from './ThirdRow';
import LastSection from './LastSection';

export {
  FirstSection,
  SecondSection,
  OurMission,
  FirstRow,
  SecondRow,
  ThirdRow,
  LastSection
}