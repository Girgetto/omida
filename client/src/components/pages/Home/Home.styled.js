import styled from 'styled-components';
import Home from './Home.component';

export default styled(Home).attrs({})`
  .firstSection {
    height: 724px;
    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;
    background-image: url(https://www.hotelseeker.it/wp-content/themes/hotelseeker/images/copertina.jpg);

    .mainTitle {
      font-size: 2rem;
      color: white;
    }

    .inputs {
      margin: 2%;
      width: 100%;
      display: flex;
      justify-content: center;
      align-items: center;
      background-color: white;
      width: 80%;
      border-radius: 5px;

      input {
        width: 60%;
        height: 50%;
        font-size: 1rem;
        background-color: #ececec;
        color: #646464;
        border-radius: 5px;
        border: 0;
        padding: 0 2%;
      }

      button {
        padding: 1% 5%;
        background-color: #2a41e8;
        color: #fff;
        border: 0;
        margin: 2%;
        padding: 2% 5%;
        border-radius: 5px;
      }
    }
  }

  .secondSection {
    text-align: center;
    margin: 5% 0%;

    .cardsContainer {
      display: flex;
      flex-wrap: wrap;

      .card {
        width: 30%;
        margin: 1%;
        position: relative;

        .card-image {
          width: 100%;
        }
      }
    }

    .all-properties-button {
      background-color: #000;
      color: #fff;
      border: 0;
      padding: 1% 5%;
      border-radius: 15px;
    }
  }
  .thirdSection {
    text-align: center;
    margin: 5% 0%;

    .content {
      padding: 10%;
      display: flex;

      .left-content {
        width: 50%;

        .deal-image {
          width: 100%;
        }
      }

      .right-content {
        width: 50%;
        display: flex;
        flex-direction: column;
        justify-content: space-around;
        align-items: center;
        background-color: #292929;

        p {
          color: white;
        }

        button {
          color: white;
          background-color: #ff5e15;
          border: 0;
          border-radius: 4px;
          padding: 0.6rem 1.8rem;
        }
      }
    }
  }

  .fourthSection {

    .main-title {
      text-align: center;
    }

    .fourth-main {
      background-image: url('https://i.ibb.co/QjLjVVx/hero-bg-image.png');
      background-repeat: no-repeat;
      width: 100%;
      background-size: 110%;
      padding: 5% 2%;
      height: 400px;

      .main-inner {
        background-color: #2842cd;
        width: 400px;
        height: 60%;
        margin: 2%;
        padding: 2%;
        display: flex;
        flex-direction: column;
        justify-content: space-around;
        align-items: flex-start;
        
        h1 {
          color: #fff;
        }

        p {
          color: #fff;
          font-size: 0.8rem;
        }

        button {
          background-color: #000;
          padding: 8px 20px;
          border-radius: 10px;
          border: none;
          color: #fff;
        }
      }
    }
  }
`;
