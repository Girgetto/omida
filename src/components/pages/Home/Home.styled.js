import styled from 'styled-components';
import Home from './Home.component';

export default styled(Home).attrs({})`
  .firstSection {
    height: 724px;
    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;
    background-color: black;

    .mainTitle {
      font-size: 46px;
      font-weight: 500;
      color: white;
    }
  }
`;
