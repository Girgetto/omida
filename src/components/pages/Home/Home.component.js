import React, { Suspense } from 'react'
import * as Comp from './components'
import { SEO } from '../../layout'

const Home = ({ className }) => {
  return (
    <Suspense fallback={<div>Loading</div>}>
      <SEO title="Omida - Real Estate Consulting - Italy | Home page" description="Remida - real estate">
        <div className={className}>
          <Comp.Main />
          <Comp.SecondSection />
          <Comp.OurMission />
          <Comp.FirstRow />
          <Comp.SecondRow />
          <Comp.ThirdRow />
          <Comp.LastSection />
        </div>
      </SEO>
    </Suspense>
  )
}

export default Home
