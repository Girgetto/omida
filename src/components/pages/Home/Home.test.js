import React from 'react';
import { render } from '@testing-library/react';
import { ParallaxProvider } from 'react-scroll-parallax'

import Home from './Home.component';

const props = {
  className: 'className',
};

describe('<Home />', () => {
  const { container } = render(
    <ParallaxProvider>
      <Home {...props} />,
    </ParallaxProvider>
  );

  it('match snapshot', () => {
    expect(container).toMatchSnapshot();
  });
});
