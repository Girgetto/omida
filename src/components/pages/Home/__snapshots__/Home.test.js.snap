// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> match snapshot 1`] = `
<div>
  <div
    class="className"
  >
    <section
      class="firstSection"
    >
      <div
        class="mainTitle"
      >
        <h1
          data-aos="fade-right"
          data-aos-duration="1000"
        >
          title
        </h1>
        <p
          data-aos="fade-left"
          data-aos-duration="1000"
        >
          home.RealEstate
        </p>
      </div>
    </section>
    <section
      class="secondSection"
    >
      <div
        class="titles"
      >
        <p
          class="paraph"
        >
          home.Analysis
           |Â 
          home.Consultancy
           |
           
          home.Coordination
        </p>
      </div>
      <p
        class="consultancy"
      >
        home.ConsultancyMediation
      </p>
    </section>
    <div
      class="ourMission"
      data-aos="fade-up"
      data-aos-anchor-placement="top-center"
    >
      <div
        class="column_2"
      >
        <div
          class="parallax-outer"
        >
          <div
            class="parallax-inner"
            style="transform: translate3d(0%, 40%, 0);"
          >
            <img
              alt="view4"
              class="image"
              src="view4.jpg"
            />
          </div>
        </div>
      </div>
      <div
        class="column_2"
      >
        <div>
          <h2>
            home.OurMission
          </h2>
        </div>
        <div>
          <p>
            home.WeOffer
          </p>
        </div>
      </div>
    </div>
    <div
      class="squares"
      data-aos="fade-up"
      data-aos-anchor-placement="bottom-bottom"
    >
      <div
        class="square wrapper"
      >
        <h2>
          home.TheExperience
        </h2>
        <p>
          home.TheKnowledge
        </p>
      </div>
      <div
        class="square"
      >
        <img
          alt="castle"
          class="image"
          src="castle.jpg"
        />
      </div>
      <div
        class="square wrapper"
      >
        <h2>
          home.WhenBeing
        </h2>
        <p>
          home.InThree
        </p>
      </div>
    </div>
    <div
      class="squares"
      data-aos="fade-up"
      data-aos-anchor-placement="top-center"
    >
      <div
        class="square"
      >
        <img
          alt="view1"
          class="image"
          src="view1.jpg"
        />
      </div>
      <div
        class="square wrapper"
      >
        <h2>
          home.ASpecializedGuide
        </h2>
        <p>
          home.WeAnalyze
        </p>
      </div>
      <div
        class="square"
      >
        <img
          alt="view2"
          class="image"
          src="view2.jpg"
        />
      </div>
    </div>
    <div
      class="squares"
      data-aos="fade-up"
      data-aos-anchor-placement="top-center"
    >
      <div
        class="square wrapper"
      >
        <h2>
          home.PrivacyAndDiscretion
        </h2>
        <p>
          home.OurApproach
        </p>
      </div>
      <div
        class="square"
      >
        <img
          alt="view3"
          class="image"
          src="view3.jpg"
        />
      </div>
      <div
        class="square wrapper"
      >
        <h2>
          home.LuxuryRealEstate
        </h2>
        <p>
          home.WithAGrowing
        </p>
      </div>
    </div>
    <div
      class="lastSection"
      data-aos="fade-up"
      data-aos-anchor-placement="bottom-bottom"
    >
      <div>
        <h2>
          home.ATradition
        </h2>
      </div>
      <div>
        <p>
          home.WeDevelop
        </p>
      </div>
    </div>
  </div>
  ,
</div>
`;
